compiler/graindoc/docblock.re

```diff
@@ -145,6 +145,49 @@ let to_markdown = docblock => {
       deprecations,
     );
   };
+  // TODO: Should we fail if more than one `@since` attribute?
```

Comment:
Probably.

Reply:
@ospencer I was just trying to implement this and I actually want to do it in a separate PR because I want to raise a good exn and it would have to exist in 2 places. I'm going to plan to refactor the module docblock stuff and the per-export docblock into deduped methods. I'll create an issue for this.

Reply:
#787